src/test/kotlin/note/application/NoteUpdaterTest.kt

```diff
@@ -0,0 +1,65 @@
+package note.application
+
+import note.domain.*
+import note.domain.exceptions.NonExistentNoteException
+import note.mothers.IdentifierMother
+import note.mothers.NoteMother
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.Mockito
+
+class NoteUpdaterTest {
+    private lateinit var noteUpdater: NoteUpdater
+    private lateinit var generator: NoteIdentifierGenerator
+    private lateinit var repository: NoteRepository
+
+    private val newTitle = "title"
```

Comment:
Use mothers instead variables here. If one day we change the domain rules we would have to change the mothers and all the remaining tests.

Encapsulating the instantiation logic inside the mothers make domain changes trivial and doesn't break the test suite.